src/handlers/app.rs

```diff
@@ -1,5 +1,10 @@
-use std::collections::{HashMap, VecDeque};
+use std::{
+    collections::{HashMap, VecDeque},
+    path::PathBuf,
+};
```

Comment:
`path::PathBuf` is never used.

```suggestion
use std::collections::{HashMap, VecDeque};
```